src/Command/BuyCommand.php

```diff
@@ -56,6 +57,18 @@ public function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        $assetToBuy = (string) $input->getArgument('asset');
+        // TODO check if this a valid token which can be bought on the exchange
+        if ($assetToBuy === '') {
```

Comment:
Ik weet niet zeker of deze situatie eigenlijk wel kan voorkomen, je hebt "asset" al als REQUIRED aangemerkt, je kan deze dus niet meegeven en toch leeg hebben.

Reply:
True, die check is niet nodig.